pallets/pools/src/lib.rs

```diff
@@ -2121,6 +2171,42 @@ pub mod pallet {
 			pool.available_reserve = pool.total_reserve;
 
 			Self::rebalance_tranches(pool, &epoch, &executed_amounts)?;
+			Self::execute_scheduled_update(pool_id, pool, Some(epoch), Some(solution))?;
```

Comment:
I would not add this to the already big epoch execution, but rather enforce anybody to call an `execute_scheduled_update` extrinsic.